site/content/en/docs/start/_index.md

```diff
@@ -1,74 +1,73 @@
 ---
-title: "minikube start"
-linkTitle: "Get Started!"
-weight: 1
-aliases:
+Titre: "Commencer Minikube"
```

Comment:
probably, just translating the `en` version of `site` to French is not what we want :)